### ISSUE TITLE
LookupError: unknown encoding: idna

### ISSUE THREAD

**Issue by az0** [bleachbit/bleachbit#760]
[Build 1495](https://ci.appveyor.com/project/az0/bleachbit/builds/30362594)
BleachBit-3.1.0.1495-portable

Windows 10 64-bit

This error happens on startup
```
Error al buscar actualizaciones:
Traceback (most recent call last):
  File "C:\projects\bleachbit\bleachbit\GUI.py", line 1080, in check_online_updates
  File "C:\projects\bleachbit\bleachbit\Update.py", line 157, in check_updates
  File "C:\Python34\lib\urllib\request.py", line 464, in open
  File "C:\Python34\lib\urllib\request.py", line 482, in _open
  File "C:\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python34\lib\urllib\request.py", line 1226, in https_open
  File "C:\Python34\lib\urllib\request.py", line 1183, in do_open
  File "C:\Python34\lib\http\client.py", line 1137, in request
  File "C:\Python34\lib\http\client.py", line 1182, in _send_request
  File "C:\Python34\lib\http\client.py", line 1133, in endheaders
  File "C:\Python34\lib\http\client.py", line 963, in _send_output
  File "C:\Python34\lib\http\client.py", line 898, in send
  File "C:\Python34\lib\http\client.py", line 1279, in connect
  File "C:\Python34\lib\http\client.py", line 871, in connect
  File "C:\Python34\lib\socket.py", line 498, in create_connection
  File "C:\Python34\lib\socket.py", line 537, in getaddrinfo
LookupError: unknown encoding: idna
```

**Comment by mkhon** [comment]
I can't reproduce this. Windows 10 x64

**Comment by az0** [comment]
Fixed with ab76ebd39616e23e34036b8c1f5602309326b40b
